Makefile

```diff
@@ -35,19 +35,25 @@ check_version_dirty:
 #-----------------
 #-- integration
 #-----------------
-.PHONY: $(BINDIR)/test integration integration.ads integration.xds integration.rest integration.ads.tls
+.PHONY: $(BINDIR)/test integration integration.ads integration.ads.v3 integration.xds integration.xds.v3 integration.rest integration.ads.tls
 
 $(BINDIR)/test:
 	@go build -race -o $@ pkg/test/main/main.go
 
-integration: integration.xds integration.ads integration.rest integration.ads.tls
+integration: integration.xds integration.xds.v3 integration.ads integration.ads.v3 integration.rest integration.ads.tls
 
 integration.ads: $(BINDIR)/test
 	env XDS=ads build/integration.sh
 
+integration.ads.v3: $(BINDIR)/test
+	env XDS=ads SUFFIX=v3 build/integration.sh
+
 integration.xds: $(BINDIR)/test
 	env XDS=xds build/integration.sh
 
+integration.xds.v3: $(BINDIR)/test
+	env XDS=xds SUFFIX=v3 build/integration.sh
+
 integration.rest: $(BINDIR)/test
```

Comment:
Curious: is REST v3 not working?

Reply:
it didn't work out in the first attempt. I will work on it in a separate PR.

Reply:
In rest integration tests, there's an error `unknown field "build_version" in envoy_config_core_v3.Node` at this [point](https://github.com/envoyproxy/go-control-plane/blob/master/pkg/server/v3/gateway.go#L78).

Im assuming this is an envoy bug since it's set in envoy source.